src/primitives/torus.jl

```diff
@@ -62,7 +62,7 @@ area(t::Torus) = measure(t)
 function Base.in(p::Point, t::Torus)
   c, n⃗ = t.center, t.normal
   R, r = t.major, t.minor
-  M = uvrotation(Vec(0, 0, 1), n⃗)
```

Comment:
I think you misunderstood my request. I was wondering if rotation_between could have a different order of arguments, our uvrotation has the expected order of arguments.

Reply:
Okay.., but is it okay to have inconsistent behavior? (https://github.com/JuliaGeometry/Meshes.jl/pull/415#discussion_r1186701645)
I was thinking that the current argument order of `uvrotation(::Vec2, ::Vec2)` is correct, but `uvrotation(::Vec3, ::Vec3)` is not.

Reply:
Sorry, maybe I misunderstood something. I thought that Rotations.rotation_between had the same purpose of Meshes.uvrotation and that we could remove our uvrotation in favor of rotation_between. I then started to wonder why rotation_between expects (v, u) instead of (u, v). Is rotation_between(v, u) the rotation necessary to move v to u?

Reply:
>Is rotation_between(v, u) the rotation necessary to move v to u?

Yes, that's right. `Rotations.rotation_between(u, v)` is a rotation that moves `u` to `v`. Here's an examlpe:

```julia
julia> using Rotations, StaticArrays

julia> u = SVector(1,2,3);

julia> v = SVector(3,1,2);

julia> rotation_between(u, v) * u ≈ v
true
```

However, `Meshes,uvroatation(u::Vec3, v::Vec3)` is a rotation that moves `v` to `u`. (NOT `u` to `v`)

```julia
julia> using Meshes

julia> u = Vec3(1,2,3);

julia> v = Vec3(3,1,2);

julia> uvrotation(u, v) * u ≈ v
false

julia> uvrotation(u, v) * v ≈ u
true
```

Note that `Meshes,uvroatation(u::Vec2, v::Vec2)` is a rotation that moves `u` to `v`.

```julia
julia> u = Vec2(1,3);

julia> v = Vec2(3,1);

julia> uvrotation(u, v) * v ≈ u
false

julia> uvrotation(u, v) * u ≈ v
true
```

Reply:
Oh then this yet another reason to eliminate our uvrotation completely from the codebase. Can we remove it and use rotation_between everywhere?

Reply:
Sorry for the late reply. I agree with removing `uvrotaiton`, and I'd like to merge https://github.com/JuliaGeometry/Rotations.jl/pull/257 first. (x-ref https://github.com/JuliaGeometry/Meshes.jl/pull/415#discussion_r1186704202)